### PR TITLE
COMMERCE-11472 same check regardless the list size

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-11472
Resent from https://github.com/liferay-commerce/liferay-portal/pull/3761

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 1]
```diff
@@ -455,6 +455,86 @@ definition {
...
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And the product of 'ABS Sensor' with an expired SKU is updated") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:478]: "And the SKU of 'ABS Sensor' is updated to be expired" would be better

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
cc/ @victoroscuro

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by victoroscuro** [comment]
ci:forward

**Pull request closed without merging**